pkg/cloud/gcp/actuators/machine/machine_scope.go

```diff
@@ -169,11 +163,29 @@ func (s *machineScope) storeMachineStatus() error {
 	s.machine.Status.ProviderStatus = ext
 	time := metav1.Now()
 	s.machine.Status.LastUpdated = &time
-	latestMachine, err := s.machineClient.UpdateStatus(s.machine)
-	if err != nil {
+
+	return nil
+}
+
+func (s *machineScope) PatchMachine() error {
+	klog.V(3).Infof("%v: patching machine", s.machine.GetName())
```

Comment:
Nit: should this be a string instead?
```suggestion
	klog.V(3).Infof("%s: patching machine", s.machine.GetName())
```